src/main/kotlin/me/alexflipnote/kawaiibot/commands/Urban.kt

```diff
@@ -44,12 +44,21 @@ class Urban : ICommand {
                     if (defs.length() == 0) {
                         return@thenAccept ctx.send("Couldn't find your search in the dictionary...")
                     }
-                    val def = defs.getJSONObject(0) // maybe random?
-                    if (def.getString("definition").length > 1024) {
-                        val permalink = def.getString("permalink")
+                    var max = defs.getJSONObject(0)
+                    var maxthumbs = max.getInt("thumbs_up")
+                    for (i in 1..(defs.length() - 1)) {
```

Comment:
Any reason why this is `for (i in 1..` and not `for (i in 0..`?

Reply:
Probably used to Lua maybe, where Array starts with 1?

Reply:
Unless my understanding is wrong, there's no need to iterate through the zeroth element as the code initially assumes the zeroth element is the maximum, so `i=0` would just compare the zeroth element with itself

Reply:
Ah yeah I see